toolchain/testing/file_test.cpp

```diff
@@ -230,6 +231,13 @@ auto ToolchainFileTest::DoExtraCheckReplacements(std::string& check_line) const
     // The column happens to be right for FileStart, but the line is wrong.
     static RE2 file_token_re(R"((FileEnd.*column: |FileStart.*line: )( *\d+))");
     RE2::Replace(&check_line, file_token_re, R"(\1{{ *\\d+}})");
+  } else if (component_ == "check") {
+    // The path to the core package appears in some check diagnostics, and will
+    // differ between testing environments, so don't test it.
+    // TODO: Consider adding a content keyword to name the core package, and
+    // replace with that instead.
```

Comment:
Had you considered instead having core added to the vfs, maybe as an overlay so they're not copied per-test, so that the driver could see consistent paths instead of needing to do regex replacements? I'm fine with that not being here, but may offer a better TODO.

Reply:
Added that to the TODO.